docs/streams/developer-guide/config-streams.html

```diff
@@ -775,6 +776,20 @@ <h4><a class="toc-backref" href="#id46">ensure.explicit.internal.resource.naming
             </div>
           </blockquote>
         </div>
+        <div class="section" id="group-protocol">
+          <h4><a class="toc-backref" href="#id47">group.protocol</a><a class="headerlink" href="#group-protocol" title="Permalink to this headline"></a></h4>
+          <blockquote>
+            <div>
+              <p>
+                The group protocol used by the Kafka Streams client used for coordination.
+                This is used to determine how the client will communicate with the Kafka brokers and other clients in the same group.
+                The default value is <code class="docutils literal"><span class="pre">"classic"</span></code>, which is the classic consumer group protocol.
+                Set to <code class="docutils literal"><span class="pre">"streams"</span></code> to enable the new Kafka Streams group protocol.
```

Comment:
```suggestion
                Can be set to <code class="docutils literal"><span class="pre">"streams"</span></code> (requires broker side enablement) to enable the new Kafka Streams group protocol.
```

Reply:
Or something like this -- maybe also a link to the details how to enable it broker side?